go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/go-logr/logr v1.4.2
 	github.com/golang/mock v1.4.4
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/open-feature/open-feature-operator/apis v0.2.41-0.20240506125212-c4831a3cdc00
+	github.com/open-feature/open-feature-operator/apis v0.2.44
```

Comment:
With the orginal go.mod, the app doesn't compile (tested on 3 different machines with an empty gomod cache)